demos/operations-workflow-for-customizing-builder-images/README.md

```diff
@@ -0,0 +1,169 @@
+# Operations Workflow for Customizing Builder Images #
+
+
+This demonstration describes how Dev code and Ops code are merged in the build pipeline on OpenShfit V3.
+
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Bill of Materials](#bill-of-materials)
+	* [Environment Specifications](#environment-specifications)
+	* [Template Files](#template-files)
+	* [Config Files](#config-files)
+	* [External Source Code Repositories](#external-source-code-repositories)
+* [Setup Instructions](#setup-instructions)
+* [Presenter Notes](#presenter-notes)
+	* [Main Demo Steps](#main-demo-steps)
+* [Resources](#resources)
+
+
+
+##Overview
+
+The demo aims to show how the Ops group can customize an external image (Red Hat EAP6) and then have that image used by development.  The following diagram shows that process at a high level.  The demo 
+![Demo Overview](images/demo-overview.png "Demo Overview")
+
+The demo also includes a basic security outline on groups can control access to both the Ops and Development projects.  The following diagram show the security model that's utilized in the demo.  
+![Security Overview](images/security-overview.png "Security Overview")
+
+*The full presentation deck is included in the Git repo.* 
+
+
+## Bill of Materials
+
+### Environment Specifications
+
+This demo should be run on an installation of OpenShift Enterprise V3
+
+### Template Files
+
+None
+
+### Config Files
+
+None
+
+### External Source Code Repositories
+
+* [Ticket Monster](https://github.com/jboss-developer/ticket-monster) -  Public repository for the Ticket Monster application, used to generate war artifact.  
+* [Ops Repo - EAP6 Customization](https://github.com/themoosman/ops-custom-eap6) -  Public repository that holds configuration information (Infrastructure as Code) used to generate (in our case) a custom EAP6 image.
```

Comment:
@themoosman I mean changing the namespace of https://github.com/themoosman/ops-custom-eap6 to https://github.com/rhtconsulting/ops-custom-eap6